api-key-thing-python/Generator.py

```
@@ -0,0 +1,25 @@
+import json, os, hashlib, secrets
+from datetime import datetime, timezone
+
+p = os.path.join(os.path.dirname(__file__), 'db.json')
+
+def get_db():
+    try:
+        with open(p) as f: return json.load(f)
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)